.github/workflows/codeql.yml

```
@@ -0,0 +1,109 @@
+name: "CodeQL Advanced"
+
+on:
+  push:
+    branches: [ "master" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)